Analysis/R/prepare_stan_input.R

```diff
@@ -484,158 +484,158 @@ prepare_stan_input <- function(
     res_space = res_space,
     sf_grid = sf_grid,
     grid_changer = grid_changer)
-  
+
   # Get pixels with low sfrac
-  output_low_sfrac <- output_location_periods_table %>% 
-    dplyr::group_by(rid, x, y) %>% 
-    dplyr::slice_max(pop_weight) %>% 
-    dplyr::filter(pop_weight < sfrac_thresh) %>% 
-    dplyr::select(rid, x, y) %>% 
+  output_low_sfrac <- output_location_periods_table %>%
+    dplyr::group_by(rid, x, y) %>%
+    dplyr::slice_max(pop_weight) %>%
+    dplyr::filter(pop_weight < sfrac_thresh_conn) %>%
```

Comment:
This should be sfrac_thresh_border.

Reply:
I’m not sure I understand why that line should be changed to sfrac_thresh_border. Isn’t the pixel linkage to an output shapefile more equivalent to the sfrac_thresh_conn setting? This isn’t going to affect the appearance of any grid cell estimate in our outputs

Reply:
I see what you mean. Also, thinking about it, if the national shapefile for observations is the same as the one for the output, then this dropping part should not be necessary, and the connection part after should take care of everything. So we can either keep here sfrac_thresh_conn as you suggest, or remove all lines 513-523.